packages/server/src/appMigrations/index.ts

```diff
@@ -10,22 +10,36 @@ export * from "./appMigrationMetadata"
 export type AppMigration = {
   id: string
   func: () => Promise<void>
+  // disabled so that by default all migrations listed are enabled
+  disabled?: boolean
 }
 
-export const getLatestMigrationId = () =>
-  MIGRATIONS.map(m => m.id)
+export function getLatestEnabledMigrationId(migrations?: AppMigration[]) {
+  const enabledMigrations: AppMigration[] = []
+  for (let migration of migrations || MIGRATIONS) {
+    // if a migration is disabled, all migrations after it are disabled
+    if (migration.disabled) {
+      break
+    }
+    enabledMigrations.push(migration)
```

Comment:
Do we need to keep pushing it? Could we not just order on the foreach and keep getting the latest it?